.gitignore

```diff
@@ -1,2 +1,4 @@
 /target
 **/*.rs.bk
+bin
```

Comment:
I'm curious about this.. does Visual Studio not build with Cargo? Normally all Rust tooling uses Cargo, which places all build artifacts inside `target`.

Reply:
That rule is not necessary actually. I pushed a new commit where it's deleted.